drivers/hmis/app/models/hmis/concerns/hmis_arel_helper.rb

```diff
@@ -64,6 +64,18 @@ def u_t
     def ut_t
       Hmis::UnitType.arel_table
     end
+
+    def c_t
+      Hmis::Hud::Client.arel_table
+    end
+
+    def p_t
+      Hmis::Hud::Project.arel_table
+    end
+
+    def o_t
+      Hmis::Hud::Organization.arel_table
+    end
```

Comment:
You shouldn't need to add any of these because we already have them from from the warehouse [ArelHelper](https://github.com/greenriver/hmis-warehouse/blob/release-80/app/models/concerns/arel_helper.rb) which is included in this one. So the HmisArelHelper just contains any "extra" HMIS-specific tables